src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Packaging/ProvideDiffSupportedContentTypeAttribute.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio
+{
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = true)]
+    internal class ProvideDiffSupportedContentTypeAttribute : RegistrationAttribute
+    {
+        private const string RegistryKey = @"Diff\SupportedContentTypes";
+
+        private readonly string _contentType;
+        private readonly string _extension;
+
+        public ProvideDiffSupportedContentTypeAttribute(string extension, string contentType)
+        {
+            _extension = extension;
+            _contentType = contentType;
+        }
+
+        public override void Register(RegistrationContext context)
+        {
+            using Key key = context.CreateKey(RegistryKey);
+
+            key.SetValue(_extension, _contentType);
+        }
+
+        //
```

Comment:
nit: remove //

Reply:
Thank you! And thanks for the reminder to log a bug with roslyn for why it inserts them when looking at a decompilation :)